powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/converter/v2/SignatureTypeConverter.java

```diff
@@ -71,7 +75,7 @@ public SignatureType convertFrom(PowerAuthSignatureTypes powerAuthSignatureTypes
             case POSSESSION_BIOMETRY:
                 return SignatureType.POSSESSION_BIOMETRY;
             default:
-                return SignatureType.POSSESSION_KNOWLEDGE_BIOMETRY;
+                return null;
```

Comment:
Are you sure this does not have catastrophic consequences somewhere?

Reply:
The `null` value is always checked and an exception is thrown.